src/action.ts

```diff
@@ -71,6 +71,15 @@ async function action(pr: PullRequest): Promise<void> {
     ? tmtArtifactsSchema.parse(tmtArtifactsParsed.data)
     : [];
 
+  // Generate tmt hardware specification
+  // See https://tmt.readthedocs.io/en/stable/spec/plans.html#hardware
+  // https://gitlab.com/testing-farm/docs/root/-/merge_requests/120/diffs?view=inline
+
+  let tmtHardware;
+  if (getInput('tmt_hardware') !== '') {
```

Comment:
I would rather store `tmt_hardware` in constant and use that for comparison.

```suggestion
  const rawTmtHardware = getInput('tmt_hardware');
  if (rawTmtHardware) {
```